flocker/node/agents/cinder.py

```diff
@@ -356,17 +373,14 @@ def create_volume(self, dataset_id, size):
             CLUSTER_ID_LABEL: unicode(self.cluster_id),
             DATASET_ID_LABEL: unicode(dataset_id),
         }
-        action_type = u"blockdevice:cinder:create_volume"
-        with start_action(action_type=action_type):
-            requested_volume = self.cinder_volume_manager.create(
-                size=int(Byte(size).to_GiB().value),
-                metadata=metadata,
-            )
-            Message.new(blockdevice_id=requested_volume.id).write()
```

Comment:
You pointed out a very valid reason to keep ``Message``, will bring it back.
``start_action`` is no longer necessary since logging decorator has ``with OPENSTACK_ACTION..``.